R/run.R

```diff
@@ -0,0 +1,69 @@
+
+#'@title Run the simulation
+#'@description parameterises and runs the model. The output will be returned
+#'@param model parameters to override
+#'@param farauti parameters to override
+#'@param punctulatus parameters to override
+#'@param koliensis parameters to override
+run_simulation <- function(
+  model = NULL,
+  farauti = NULL,
+  punctulatus = NULL,
+  koliensis = NULL
+  ) {
+  basedir <- system.file('defaults', package = 'vivax')
```

Comment:
The option `mustWork = TRUE` is useful in all uses of `system.file`, unfortunately